app/presenters/content_item/links_out.rb

```diff
@@ -0,0 +1,107 @@
+module ContentItem
+  module LinksOut
+    def links_out_supergroups
```

Comment:
Should all these methods be added to the public interface of the class that includes this module?